bundle/manifests/kuadrant.io_dnshealthcheckprobes.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
are these copies? Do we need to check them in or could they be copied from the config/crd location at build time?

Reply:
Yes they get generated at build time for the bundle from the config/crd, I can take them out but a good few other repos keep them in like Limitador and kuadrant